ossfuzz/sndfile_fuzz_header.h

```diff
@@ -0,0 +1,114 @@
+typedef struct
```

Comment:
I'd have expected to see an ifdef guard here, e.g. https://github.com/libsndfile/libsndfile/blob/master/src/mpeg.h#L20

```
#ifndef SNDFILE_FUZZ_HEADER_H
#define SNDFILE_FUZZ_HEADER_H
```
with the closing `#endif` at the end.